### PR TITLE
fix slice copy in 1.20

### PR DESCRIPTION
**Pull request by tylitianrui**
the Slices package is  added  in Go 1.21. so  `Compilation failed` is  caused by  importing  package slices  in 1.20

### CODE REVIEW

File: model/histogram/generic_test.go [hunk 1]
```diff
@@ -15,7 +15,6 @@ package histogram
 
 import (
 	"math"
-	"slices"
```
[the remark this answers is not in the document]
Comment (tylitianrui) [reply model/histogram/generic_test.go:18]: thx for your review

**Review by linasm — APPROVED** [review]
Thank you, and sorry for trouble. I believe one of maintainers will have to stamp this.

**Review by machine424 — APPROVED** [review]
LGTM

There is a `test_golang_oldest` job that is supposed to run tests on 1.20, but it only seems to be running tsdb tests, I asked about it here: https://cloud-native.slack.com/archives/C01AUBA4PFE/p1705334735788509

I opened https://github.com/prometheus/prometheus/pull/13403 to migrate into `slices` in the future.

File: model/histogram/generic_test.go [hunk 2]
```diff
@@ -15,9 +15,10 @@ package histogram
...
 	"testing"
 
+	"golang.org/x/exp/slices"
+
```
[the remark this answers is not in the document]
Comment (tylitianrui) [reply model/histogram/generic_test.go]: thx for  review

**Comment by tylitianrui** [comment]
> LGTM
> 
> There is a `test_golang_oldest` job that is supposed to run tests on 1.20, but it only seems to be running tsdb tests, I asked about it here: https://cloud-native.slack.com/archives/C01AUBA4PFE/p1705334735788509
> 
> I opened #13403 to migrate into `slices` in the future.



> LGTM
> 
> There is a `test_golang_oldest` job that is supposed to run tests on 1.20, but it only seems to be running tsdb tests, I asked about it here: https://cloud-native.slack.com/archives/C01AUBA4PFE/p1705334735788509
> 
> I opened #13403 to migrate into `slices` in the future.
 
sorry  i am not in  workspace `cloud-native.slack.com`.  could you  invite me ?  thanks

**Comment by dgl** [comment]
https://slack.cncf.io/ will get you an invite.

**Review by bboreham — APPROVED** [review]
Thanks!

**Pull request merged**